### DISCUSSION TITLE
Generate qodana.yaml to match set of custom IntelliJ code inspections

### DISCUSSION THREAD

**Discussion by Gonzalo-bia** [JetBrains/Qodana#224] [Q&A] [answered]
Hello,

I'd like to generate a `qodana.yaml` file that exactly matches my custom IntelliJ code inspections file. Is there a simple way to achieve this?

Is there a way to know the exact set of inspections that are included in each predefined Qodana profile?

Greetings
Thanks

**Comment by Gonzalo-bia** [comment]
can anybody help please?

**Comment by brichbash** [comment, marked as answer]
Hi,
There is no way to convert your intellij profile into a yaml one. However, you can just export your [XML IntelliJ profile](https://www.jetbrains.com/help/qodana/custom-xml-profiles.html) and use it in Qodana.
As for the exact set of inspections in the bundled profiles. Besides, the **Configuration** tab in your Qodana reports, you can find this information in [our public repository](https://github.com/JetBrains/qodana-profiles/tree/233/.idea/inspectionProfiles) with the default profiles. The `233` branch is actual for the current stable versions, the next version profiles will be under the `241` branch
